src/spinnaker-dependencies.template

```diff
@@ -10,7 +10,8 @@ versions:
   astyanax: "3.8.0"
   aws: "1.11.28"
   awsObjectMapper: "1.11.28"
-  batch: "{{org.springframework.batch:spring-batch-core}}"
+  #batch bom: "{{org.springframework.batch:spring-batch-core}}"
+  batch: "3.0.4.RELEASE"
```

Comment:
really don't want to mess with this guy at the moment hopefully we will just be done with it entirely before we change its version